### DISCUSSION TITLE
Lint Flutter ?

### DISCUSSION THREAD

**Discussion by adriens** [lschaeffer313/domaine-nc-mobile#141] [Ideas]
@lschaeffer313 , tu lintes Le source flutter au fait ?

- https://medium.com/flutter-senior/using-dart-linter-for-cleaner-code-from-config-to-github-workflow-d17e2d5f5997
- https://dart.dev/guides/language/analysis-options

**Comment by lschaeffer313** [comment]
Nope mais il faudrait
Je vais le rajouter la prochaine fois que je mets dessus.

**Reply by adriens** [reply]
Ok, je transforme en issue et je charge sur la Roadmap :zap:
